test/Makefile

```diff
@@ -20,11 +20,10 @@ GO_VALIDATION_ERR = Go version is not supported. Please update to at least $(MIN
 
 all: $(TEST_BINARIES) phaul-test
 	mkdir -p image
-	PID=$$(piggie/piggie) && { \
+	PID=$$(piggie/piggie) && \
+	trap 'pkill -9 piggie' INT EXIT TERM && \
```

Comment:
Ah, just had a look here. Instead of doing a pkill we could also do a kill on $PID, right? Less ambiguous. Instead of killing all processes with that name we would just kill the one.

Reply:
Nice catch!